cmd/ingress-shim/controller/sync.go

```diff
@@ -20,6 +20,9 @@ const (
 	// the default configuration provided to ingress-annotation should be
 	// created.
 	tlsACMEAnnotation = "kubernetes.io/tls-acme"
+	// editInPlaceAnnotation is used to toggle the use of ingressClass instead
+	// of ingress on the created Certificate resource
+	editInPlaceAnnotation = "certmanager.k8s.io/edit-in-place"
```

Comment:
We should probably prefix this with `acme-http01` to bring it in-line with other annotations, e.g. `certmanager.k8s.io/acme-http01-edit-in-place`. It's quite wordy, but it *is* more specific (and I'd hope more obvious to the user what it is for?)